lib/index.js

```diff
@@ -1,11 +1,16 @@
 const parser = require('scratch-parser');
 
+const sb1 = require('./sb1');
 const sb2 = require('./sb2');
 const sb3 = require('./sb3');
 
 module.exports = function (buffer, callback) {
     parser(buffer, false, (err, result) => {
-        if (err) return callback(err);
+        if (err === 'Parser only supports Scratch 2.X and above') {
+            return sb1(buffer, callback);
```

Comment:
Am I reading this correctly that the SB1 path never ends up touching `scratch-parser`? Are `scratch-parser` and `scratch-sb1-converter` guaranteed to have the same output format?

Reply:
@cwillisf  Correct, the [object returned by `scratch-sb1-converter`](https://github.com/scratchfoundation/scratch-sb1-converter/blob/develop/src/to-sb2/json-generator.js#L211) seems to have the same structure as an SB2 project json, but is never validated by `scratch-parser`, and I think we shouldn't rely on it to be always valid.

I modified the code, so that the conversion result is validated using the `validate` module from `scratch-parser`.